### PR TITLE
🧼 Upgrade and clean yarn dependencies

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?

The aim of this PR is to clean our `package.json` and the associated warnings we add when installing packages. 

#### What should we test?
Green build. Probably worth a deployment on staging.

#### Release notes

Changelog Category: Technical changes

### CODE REVIEW

**Comment by mkllnk** [comment]
Needs rebase.

**Comment by jibees** [comment]
> Needs rebase.

Done.

File: package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.20.5",
+    "@babel/plugin-transform-runtime": "^7.19.6",
```
Comment (mkllnk) [inline package.json:46]: I really like when commit messages are explaining the why instead of the what.

**Review by mkllnk — APPROVED** [review]
Thank you!

**Comment by filipefurtad0** [comment]
Hey @jibees ,

Unfortunately, it seems deployment is failing for this one, for details see:
https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/servers/au-staging/deploys/545

It seems there is a `Compilation failed` error found in the output above. Staging master works fine, so it looks like it is really introduced by this PR. Moving back to in Dev.

**Comment by jibees** [comment]
We probably should solve https://github.com/openfoodfoundation/openfoodnetwork/pull/10398 before.
+ then we will resolve warnings.

Closing then.

**Pull request closed without merging**